src/libraries/Microsoft.PowerFx.Connectors/ODataQueryableTableValue.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Specialized;
+using System.Web;
+using Microsoft.PowerFx.Core.IR;
+using Microsoft.PowerFx.Types;
+
+namespace Microsoft.PowerFx.Connectors
+{
+    public abstract class ODataQueryableTableValue : QueryableTableValue
+    {
+        private readonly Uri _uriBase;
+        internal readonly NameValueCollection ODataParams;
```

Comment:
>NameValueCollection [](http://example.com/codeflow?start=26&length=19)

Is this like $filter, $select, etc? 
Could we make this strongly typed, or is it intrinsically dynamic?

Reply:
Added struct type